rest/rest-api/build.gradle

```diff
@@ -3,4 +3,6 @@ dependencies {
     compile 'org.jboss.resteasy:resteasy-multipart-provider:3.0.26.Final'
     compile "org.objectweb.proactive:programming-util:${programmingVersion}"
     compile project(':scheduler:scheduler-api')
+    compile "org.objectweb.proactive:programming-core:${programmingVersion}"
+    compile project(':rm:rm-client')
```

Comment:
these two imports are really problematic because they add a huge set of jars to all projects using rest-api (and there are a lot)

Reply:
Then I would need to move all these classes to rest-api:
import org.ow2.proactive.resourcemanager.common.NSState;
import org.ow2.proactive.resourcemanager.common.RMState;
import org.ow2.proactive.resourcemanager.common.event.RMNodeHistory;
import org.ow2.proactive.resourcemanager.common.event.RMNodeSourceEvent;
import org.ow2.proactive.resourcemanager.common.event.dto.RMStateDelta;
import org.ow2.proactive.resourcemanager.common.event.dto.RMStateFull;
import org.ow2.proactive.resourcemanager.exception.RMException;
import org.ow2.proactive.resourcemanager.frontend.topology.Topology;
import org.ow2.proactive.resourcemanager.nodesource.common.NodeSourceConfiguration;
import org.ow2.proactive.resourcemanager.nodesource.common.PluginDescriptor;